02-peer-protocol.md

```diff
@@ -580,10 +580,10 @@ shutdown starts, the question of how to behave if it wasn't is avoided:
 the sender always sends a `commitment_signed` first.
 
 As shutdown implies a desire to terminate, it implies that no new
-HTLCs will be added or accepted.  Once any HTLCs are cleared, the peer
+HTLCs will be added or accepted.  Once any HTLCs are cleared, there are no dangling commitments and all updates are included on both commitment transactions, the peer
 may immediately begin closing negotiation, so we ban further updates
 to the commitment transaction (in particular, `update_fee` would be
-possible otherwise).
+possible otherwise). However, while there are HTLCs on the commitment transaction, the initiator may find it desirable to increase the feerate as there may be pending HTLCs on the commitment which could timeout.
```

Comment:
```suggestion
possible otherwise). However, while there are HTLCs on the commitment 
transaction, the initiator may find it desirable to increase 
the feerate as there may be pending HTLCs on the commitment which could timeout.
```